### PR TITLE
4.8.5.6 - v4 Migration Cleanup

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR is part of issue #84 

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix linting
- Fix links

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.5.6_Testing_for_NoSQL_Injection.md [hunk 1]
```diff
@@ -48,35 +48,27 @@ Even if the input used within queries is completely sanitized or parameterized,
 
 For example within MongoDB, the `$where` syntax itself is a reserved query operator. It needs to be passed into the query exactly as shown; any alteration would cause a database error. However, because `$where` is also a valid PHP variable name, it may be possible for an attacker to insert code into the query by creating a PHP variable named `$where`. The PHP MongoDB documentation explicitly warns developers:
 
-    Please make sure that for all special query operators (starting with $) you use single quotes so that PHP doesn't try to replace "$exists" with the value of the variable $exists.
+`Please make sure that for all special query operators (starting with $) you use single quotes so that PHP doesn't try to replace "$exists" with the value of the variable $exists.`
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.5.6_Testing_for_NoSQL_Injection.md]: This should be more a block quote than enclosed by backticks, no? Maybe only use special ticks for variables and the $ sign

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.5.6_Testing_for_NoSQL_Injection.md [hunk 2]
```diff
@@ -48,35 +48,27 @@ Even if the input used within queries is completely sanitized or parameterized,
...
-Injection payload wordlist with examples of NoSQL Injection for MongoDB - <https://github.com/cr0hn/nosqlinjection_wordlists>
-
-**Whitepapers**\
-Bryan Sullivan from Adobe: “Server-Side JavaScript Injection” - <https://media.blackhat.com/bh-us-11/Sullivan/BH_US_11_Sullivan_Server_Side_WP.pdf>
-
-Bryan Sullivan from Adobe: “NoSQL, But Even Less Security” - <http://blogs.adobe.com/asset/files/2011/04/NoSQL-But-Even-Less-Security.pdf>
-
-Erlend from Bekk Consulting: “\[Security\] NOSQL-injection” - <http://erlend.oftedal.no/blog/?blogid=110>
+One way to potentially assign data to PHP variables is via HTTP Parameter Pollution (see: [Testing for HTTP Parameter pollution(OTG-INPVAL-004)](https://www.owasp.org/index.php/Testing_for_HTTP_Parameter_pollution_(OTG-INPVAL-004)). By creating a variable named `$where` via parameter pollution, one could trigger a MongoDB error indicating that the query is no longer valid. Any value of `$where` other than the string “\$where” itself, should suffice to demonstrate vulnerability. An attacker would develop a full exploit by inserting the following: “`$where:`` ``function()`` ``{`` ``//arbitrary`` ``JavaScript`` ``here`` ``}`”
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.5.6_Testing_for_NoSQL_Injection.md]: Please clean up this sentence. You missed a `$where`, and the last line in it

**Comment by rejahrehim** [comment]
@ThunderSon Fixed. Please check

**Comment by rejahrehim** [comment]
@ThunderSon Done

**Pull request merged**